app/models/bookable_slot.rb

```diff
@@ -23,7 +24,14 @@ def self.without_appointments
               appointments.end_at = #{quoted_table_name}.end_at
             SQL
          )
-      .where('appointments.start_at IS NULL')
+      .where(<<-SQL
```

Comment:
```ruby
where(start_at: nil)
  .or(where(status: %i(cancelled_by_customer cancelled_by_pension_wise)))
```
Maybe that's a little clearer, although not sure it would work.